homeassistant/components/flexpool/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Device is already configured"
```

Comment:
Use the common string here. See [documentation](https://developers.home-assistant.io/docs/internationalization/core/#introducing-new-strings).